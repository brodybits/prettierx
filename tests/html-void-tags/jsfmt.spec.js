run_spec(__dirname, ["html"]);
run_spec(__dirname, ["html"], { htmlVoidTags: true });
