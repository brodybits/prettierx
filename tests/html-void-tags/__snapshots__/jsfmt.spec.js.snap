// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input.html 1`] = `
====================================options=====================================
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text" />

================================================================================
`;

exports[`input.html 2`] = `
====================================options=====================================
htmlVoidTags: true
parsers: ["html"]
printWidth: 80
                                                                                | printWidth
=====================================input======================================
<input type='text'>

=====================================output=====================================
<input type="text">

================================================================================
`;
